index.html

```diff
@@ -34,11 +34,30 @@ <h1 class="text-uppercase">Philip Gilbert</h1>
         </div>
     </section>
 
-    <section id="details">
-        <div class="container">
-            DETAILS
-        </div>
-    </section>
+    <!-- <section id="details"> -->
+        <!-- <div class="container"> -->
+            <section class="details">
+                <div class="container">
```

Comment:
Nereikia turėti dviejų eillučių, kurios vienu metu ir veikiančios, ir užkomentuotos.
Ištrinti užkomentuotasias 👍

Reply:
No comments! 

Already!

"Just Do It." - Nike.